StylableSwiftUI/Classes/Stylist+Theme.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Stylist+Theme.swift
+//  StylableSwiftUI
+//
+//  Created by Kerr Marin Miller on 16/07/2021.
+//
+
+import Foundation
+
+public struct Theme: Equatable, Hashable {
+    public let name: String
+
+    static let identifierPrefix = "@"
+
+    var value: String {
+        return "\(Self.identifierPrefix)\(name)"
+    }
+
+    public init(name: String) {
+        if name.starts(with: "@") {
```

Comment:
Should this be `if name.starts(with: Theme.identifierPrefix) {`?

Reply:
It absolutely should